build/elevated-install.ps1

```diff
@@ -119,10 +112,22 @@ function Install-Kernel {
   }
 }
 
-if (-Not $DryRun) {
-  Restart-ScriptForElevation $args
+# .SYNOPSIS
+# Installs Rancher Desktop Privileged Service
+function Install-PrivilegedService {
+  Param([String] $InstallDir)
+  Write-Output "Installing Rancher Desktop Privileged Service"
+  $ExecPath = (Join-Path $InstallDir 'resources\win32\internal\privileged-service.exe')
+  $InstallCommand = "`"${InstallDir}`" install"
+  $Exec = Invoke-Expression -Command $InstallCommand
+  if ($Exec.ExitCode -ne '0') {
+    return 102
+  }
```

Comment:
>     PS C:\Users\Admin\rd\resources\win32\internal> .\privileged-service.exe install ; echo $LASTEXITCODE
>     Error: service [RancherDesktopPrivilegedService] already exists: The specified service already exists.
>     Usage:
>       privileged-service install [flags]
>     
>     Flags:
>       -h, --help   help for install
>     
>     1
We don't actually allow installing it twice — we probably should (and clobber the existing install?)

Reply:
Sure, I will make a separate PR for this one. We can merge that one first and this PR. WDYT?